evm_loader/program/src/config.rs

```diff
@@ -0,0 +1,183 @@
+//! CONFIG MODULE
+use cfg_if::cfg_if;
+
+use evm::{ U256 };
+
+cfg_if! {
+    if #[cfg(feature = "mainnet")] {
+
+        const CHAIN_ID: u64 = 111;
```

Comment:
Use next values for `CHAIN_ID`:
- 245022926 - devnet
- 245022934 - mainnet
- 245022940 - testnet

Reply:
Fixed